src/components/AssetsPage/index.jsx

```diff
@@ -44,63 +37,41 @@ export class AssetsPage extends React.Component {
     }
   }
 
-  hasNoAssets = assetsList => (
-    assetsList.length === 0
-  )
+  getPageType = (props) => {
+    if (props.assetsList.length > 0) {
+      return pageTypes.types.NORMAL;
+    } else if (Object.keys(props.request.assetTypes).length === 0) {
```

Comment:
This condition assumes props.request.assetTypes exists. Should I make this more robust by checking `'assetTypes' in props.request` first? The default request state in the reducer is the assetTypes object, so that key always exists, but only when the React component is tied to our particular Redux state. Just wanted to check!